pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   bip39: ^1.0.6
   convert: ^3.0.0
   cryptography: ^2.0.1
+  ed25519_edwards: ^0.1.0
```

Comment:
Looks like it's not used.

Reply:
I tried to use it, but it didn't do what we wanted. I thought pub remove would remove it ...